ci-operator/step-registry/baremetal/lab/agent/install/baremetal-lab-agent-install-commands.sh

```diff
@@ -137,22 +137,24 @@ function reset_host() {
   local bmc_user="${2}"
   local bmc_pass="${3}"
   local vendor="${4:-ampere}"
+  ipmi_boot_selection=$([ "${BOOT_MODE}" == "pxe" ] && echo force_pxe || echo force_cdrom)
+  sushy_boot_selection=$([ "${BOOT_MODE}" == "pxe" ] && echo PXE || echo VCD-DVD)
   echo "Resetting the host ${bmc_address}..."
   case "${vendor}" in
-    ampere)
-      ipmitool -I lanplus -H "$bmc_address" \
-        -U "$bmc_user" -P "$bmc_pass" \
-        chassis bootparam set bootflag "force_cdrom" options=PEF,watchdog,reset,power
+  ampere)
```

Comment:
Could you restore the indentation and limit the changes only to what's new? This is to avoid mixing 'refactoring' and 'feature' changes in the same PR?

Reply:
Done